pr-data.csv

```diff
@@ -630,16 +766,16 @@ https://github.com/dropwizard/dropwizard,07dfaed697427e208d65049f80a5d1949833b7c
 https://github.com/dropwizard/dropwizard,07dfaed697427e208d65049f80a5d1949833b7cd,dropwizard-configuration,io.dropwizard.configuration.JsonConfigurationFactoryTest.printsDidYouMeanOnUnrecognizedField,ID,,,
 https://github.com/dropwizard/dropwizard,07dfaed697427e208d65049f80a5d1949833b7cd,dropwizard-configuration,io.dropwizard.configuration.YamlConfigurationFactoryTest.printsDidYouMeanOnUnrecognizedField,ID,,,
-https://github.com/dropwizard/dropwizard,07dfaed697427e208d65049f80a5d1949833b7cd,dropwizard-jackson,io.dropwizard.jackson.JacksonTest.objectMapperSerializesNullValues,ID,,,
-https://github.com/dropwizard/dropwizard,07dfaed697427e208d65049f80a5d1949833b7cd,dropwizard-jersey,io.dropwizard.jersey.errors.LoggingExceptionMapperTest.handlesMethodNotAllowedWithHeaders,ID,,,
+https://github.com/dropwizard/dropwizard,616ed866460ca909cb30a4eafb786c6040218ae8,dropwizard-jackson,io.dropwizard.jackson.JacksonTest.objectMapperSerializesNullValues,ID,Rejected,https://github.com/dropwizard/dropwizard/pull/3284,
+https://github.com/dropwizard/dropwizard,616ed866460ca909cb30a4eafb786c6040218ae8,dropwizard-jersey,io.dropwizard.jersey.errors.LoggingExceptionMapperTest.handlesMethodNotAllowedWithHeaders,ID,,,
 https://github.com/dropwizard/dropwizard,07dfaed697427e208d65049f80a5d1949833b7cd,dropwizard-jersey,io.dropwizard.jersey.jackson.JacksonMessageBodyProviderTest.returnsValidatedSetRequestEntities,ID,,,
 https://github.com/dropwizard/dropwizard,07dfaed697427e208d65049f80a5d1949833b7cd,dropwizard-logging,io.dropwizard.logging.DefaultLoggingFactoryPrintErrorMessagesTest.testLogbackStatusPrinterPrintStreamIsRestoredToSystemOut,OD-Vic,Accepted,https://github.com/dropwizard/dropwizard/pull/2550,
 https://github.com/dropwizard/dropwizard,07dfaed697427e208d65049f80a5d1949833b7cd,dropwizard-logging,io.dropwizard.logging.DefaultLoggingFactoryTest.testConfigure,NOD;OD-Vic,,,
-https://github.com/dropwizard/dropwizard,07dfaed697427e208d65049f80a5d1949833b7cd,dropwizard-json-logging,io.dropwizard.logging.json.layout.JsonFormatterTest.testPrettyPrintNoLineSeparator,ID,,,
-https://github.com/dropwizard/dropwizard,07dfaed697427e208d65049f80a5d1949833b7cd,dropwizard-json-logging,io.dropwizard.logging.json.layout.JsonFormatterTest.testPrettyPrintWithLineSeparator,ID,,,
+https://github.com/dropwizard/dropwizard,616ed866460ca909cb30a4eafb786c6040218ae8,dropwizard-json-logging,io.dropwizard.logging.json.layout.JsonFormatterTest.testPrettyPrintNoLineSeparator,ID,Rejected,https://github.com/dropwizard/dropwizard/pull/3284,
+https://github.com/dropwizard/dropwizard,616ed866460ca909cb30a4eafb786c6040218ae8,dropwizard-json-logging,io.dropwizard.logging.json.layout.JsonFormatterTest.testPrettyPrintWithLineSeparator,ID,Rejected,https://github.com/dropwizard/dropwizard/pull/3284,
```

Comment:
Based on the comment of the developer in this PR, I would say that your PR was actually Accepted instead of Rejected. I'll merge and make this change myself

> @contextshuffling While I'm not sure this was a real problem (since the respective tests never failed locally or in CI builds), I've changed the tests **accordingly**. Thanks for the hint!